public/resources/stylesheets/page/_admin.scss

```diff
@@ -61,6 +68,7 @@
 .content-table tbody tr:nth-of-type(even) {
   background-color: #f3f3f3;
```

Comment:
save the colors in variables, and check if that color are not already in the variables file